src/dyn/libdyn.cpp

```diff
@@ -257,6 +258,7 @@ void export_dyn_variables_objects(){
       .def("allocate_tcnbra", &dyn_variables::allocate_tcnbra)
       .def("allocate_mqcxf", &dyn_variables::allocate_mqcxf)
       .def("allocate_qtsh", &dyn_variables::allocate_qtsh)
+      .def("allocate_kcrpmd", &dyn_variables::allocate_kcrpmd)

```

Comment:
same here - you'll need to define the `allocate_kcrpmd` function in the dyn_variables class before you expose it to Python

Reply:
The allocate_kcrpmd() function was added to the dyn_variables files in the previous pull requests.